.circleci/config.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       - checkout
       - run: git submodule update --init
       # Install dependencies.
+      - run: sudo gem install jazzy --no-document --version 0.9.4 # Just for testing, remove me!
```

Comment:
Can you remove this for build stage? I think thats what you also have in comments.

Reply:
That's the plan, just needed to put it in so that the PR would actually run this step.